### PR TITLE
support customizable progress bar's color and height

### PR DESCRIPTION
**Pull request by taquangtrung**
Hi,

The current height of the progress bar is quite small (2) and its color is the same as text color, so it's difficult to see.

In this PR, I make it customizable, increase the default height to 5 and change the color to orange so that it is easier to see.

Please consider merging it.

Thank you!

### CODE REVIEW

File: eaf-browser.el [hunk 1]
```diff
@@ -150,6 +150,14 @@ and will re-open them when calling `eaf-browser-restore-buffers' in the future s
   ""
   :type 'string)
 
+(defcustom eaf-browser-progress-bar-height "5"
```
Comment (manateelazycat) [inline eaf-browser.el:153]: Default use 2, too height is ugly.

File: eaf-browser.el [hunk 2]
```diff
@@ -150,6 +150,14 @@ and will re-open them when calling `eaf-browser-restore-buffers' in the future s
...
+  ""
+  :type 'int)
+
+(defcustom eaf-browser-progress-bar-color "#ff8103"
```
Comment (manateelazycat) [inline eaf-browser.el:157]: Use foreground_color default, if user's theme is same as background, user can customize option, please don't use constant value.

**Review by manateelazycat — CHANGES_REQUESTED** [review]

**Comment by taquangtrung** [comment]
@manateelazycat : Thanks! I just updated the code

File: eaf-browser.el [hunk 3]
```diff
@@ -150,6 +150,20 @@ and will re-open them when calling `eaf-browser-restore-buffers' in the future s
   ""
   :type 'string)
 
+(defcustom eaf-browser-progress-bar-height "2"
+  ""
+  :type 'int)
+
+(defcustom eaf-browser-progress-bar-color "default"
```
Comment (manateelazycat) [inline eaf-browser.el:157]: No, Use (eaf-get-theme-foreground-color)

File: eaf-browser.el [hunk 4]
```diff
@@ -150,6 +150,20 @@ and will re-open them when calling `eaf-browser-restore-buffers' in the future s
...
+  "Possible values are `default' or a hex code `#hhhhhh' of a color.
+If it is set to `default', then the default theme foreground color
+(set by `eaf-emacs-theme-foreground-color') will be used."
+  :type 'string)
+
+(defcustom eaf-browser-text-selection-color "#ffb800"
```
Comment (manateelazycat) [inline eaf-browser.el:163]: What use of this color?

**Review by manateelazycat — CHANGES_REQUESTED** [review]

**Comment by taquangtrung** [comment]
@manateelazycat : I just updated the code. Can you double-check?

**Comment by manateelazycat** [comment]
Cool, thanks for patch. ;)

**Pull request merged**